scala3doc/src/dotty/dokka/tasty/BasicSupport.scala

```diff
@@ -44,6 +44,12 @@ trait BasicSupport:
     def getAnnotations(): List[Annotation] =
       sym.annotations.filterNot(_.symbol.packageName.startsWith("scala.annotation.internal")).map(parseAnnotation).reverse
 
+    def isDeprecated(): Option[Annotation] =
+      sym.annotations.find { a =>
+        a.symbol.packageName == "scala" && a.symbol.className.fold(false)(_ == "deprecated") ||
```

Comment:
`fold(false)(_ == "deprecated")` can be replaced with '`.contains("deprecated")"`

Reply:
Changes